zipline/algorithm.py

```diff
@@ -907,6 +940,7 @@ def _calculate_order_value_amount(self, asset, value):
         return value / (last_price * value_multiplier)
 
     @api_method
+    @check_asset_lifetime
     def order(self, asset, amount,
```

Comment:
It looks like we updated just this method to have the first parameter be named `asset` on this branch.  The other `order_*` methods still take a parameter named `sid`.  We should probably pick one or the other.